src/components/utils/useButtonHandlers.ts

```diff
@@ -0,0 +1,18 @@
+import {KeyboardEvent, KeyboardEventHandler, useCallback} from 'react';
+
+interface UseButtonHandlers<T> {
+    onKeyDown: KeyboardEventHandler<T>;
+}
+
+export function useButtonHandlers<T>(onClick?: (...args: any[]) => any): UseButtonHandlers<T> {
```

Comment:
There is nothing about button, maybe better `useActionKeyDown`? Also arg should be called `callback` of smth